public/components/custom_panels/helpers/utils.tsx

```diff
@@ -399,7 +396,7 @@ export const renderCatalogVisualization = async ({
     layoutConfig: {
       height: 390,
       margin: { t: 5 },
-      legend: { orientation: 'h', yanchor: 'top', x: 0.0, y: -0.4 },
+      legend: { visible: false },
```

Comment:
why is the legend being removed here?

Reply:
Making a call for 2.11 to remove multi-line.   The resulting legend is a single entry of `{}`, which is not helpful.
So, just removing it.